plugin/src/main/java/at/helpch/chatchat/config/holder/MessagesHolder.java

```diff
@@ -33,6 +33,7 @@ public final class MessagesHolder {
     private Component socialSpyDisabled = text("Social spy disabled", RED);
     private Component ignoredPlayer = text("Successfully ignored <player>.", GREEN);
     private Component unignoredPlayer = text("Successfully un-ignored <player>.", GREEN);
+    private Component cannotIgnoreYourself = text("You cannot ignore yourself!", RED);
```

Comment:
Name this `cantIgnoreYourself` to maintain parity with the other similar messages.